backend/scheduler-svc/causelist-document-persister.yml

```
@@ -7,10 +7,12 @@ serviceMaps:
-        - query: INSERT INTO cause_list_document (file_store_id, court_id, judge_id,hearing_date) VALUES (?, ?, ?, ?);
+        - query: INSERT INTO cause_list_document (file_store_id, court_id, judge_id,hearing_date, created_time, tenant_id) VALUES (?, ?, ?, ?, ?, ?);
           basePath: $
           jsonMaps:
             - jsonPath: $.fileStoreId
             - jsonPath: $.courtId
             - jsonPath: $.judgeId
-            - jsonPath: $.hearingDate
\ No newline at end of file
+            - jsonPath: $.hearingDate
+            - jsonPath: $.createdTime
+            - jsonPath: $.tenantId
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)